src/irmin-pack/unix/gc.ml

```diff
@@ -267,6 +272,16 @@ module Make (Args : Gc_args.S) = struct
                     stats];
                 let () = Lwt.wakeup_later t.resolver (Ok stats) in
                 Ok (`Finalised stats)
+            | `Success, `Archive, Ok _ ->
+                (* TODO actually archive garbage into lower volume *)
```

Comment:
Yip, to be clear, this is A, B, and C in https://hackmd.io/jRgzxlL4Qry6SD3k4A_Z2A#Updates-to-GC-API-and-process